MediaBrowser.Providers/TV/TheTVDB/TvDbClientManager.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using MediaBrowser.Controller.Library;
+using TvDbSharper;
+
+namespace MediaBrowser.Providers.TV
+{
+    public sealed class TvDbClientManager
+    {
+        private static volatile TvDbClientManager instance;
+        private static readonly object syncRoot = new object();
+        private static TvDbClient tvDbClient;
+        private static DateTime tokenCreatedAt;
+
+        private TvDbClientManager()
+        {
+            tvDbClient = new TvDbClient();
+            tvDbClient.Authentication.AuthenticateAsync(TVUtils.TvdbApiKey);
+            tokenCreatedAt = DateTime.Now;
+        }
+
+        public static TvDbClientManager Instance
```

Comment:
hmm, are you telling me that C# has no means of making a singleton, so you have to roll your own?

Reply:
I lifted it straight from an MSDN example, so dunno

Reply:
Just to follow up. Once #801 is merged, I'll use DI Singleton instead of a static class.